lib/ansible/plugins/connection/network_cli.py

```diff
@@ -539,9 +539,15 @@ def _sanitize(self, resp, command=None):
         '''
         cleaned = []
         for line in resp.splitlines():
-            if (command and line.strip() == command.strip()) or self._matched_prompt.strip() in line:
+            if command and line.strip() == command.strip():
                 continue
-            cleaned.append(line)
+            ignore_line = False
+            for prompt in self._matched_prompt.strip().splitlines():
+                if prompt.strip() in line:
+                    ignore_line = True
+                    break
+            if not ignore_line:
```

Comment:
```suggestion
            else:
```

I'm trying to make sure I understand what's happening here, will this change have the intended effect?

Reply:
Yes `else` will have the same intended effect. Fixed your review comment. Thanks!